pages/dashboard/index.js

```diff
@@ -34,6 +38,37 @@ module.exports = settings => {
     next();
   });
 
+  app.get('/', (req, res, next) => {
+    const adminEsts = req.user.profile.establishments.filter(est => {
+      if (est.role === 'admin' || userIsNtcoAtEst(req.user.profile, est.id)) {
+        return true;
+      }
+    });
+    Promise.all(
+      adminEsts.map(est => {
+        return Promise.all([
+          req.api(`/establishment/${est.id}/pils/reviews?status=due&onlymeta=true`),
+          req.api(`/establishment/${est.id}/pils/reviews?status=overdue&onlymeta=true`)
+        ])
+          .then(response => {
+            const due = response[0].json.meta.count;
+            const overdue = response[1].json.meta.count;
+            return {
+              estId: est.id,
+              name: est.name,
+              overdue,
+              due
+            };
+          });
+      })
+    )
+      .then(reviews => {
+        res.locals.static.adminPilReviewsRequired = reviews.filter(r => r.overdue > 0 || r.due > 0);
+      })
+      .then(() => next())
+      .catch(next);
```

Comment:
```suggestion
      .catch(err => {
        req.log('error', { message: e.message, stack: e.stack, ...e });
        // don't block dashboard rendering for failed PIL review lookup
        next();
      });
```

Reply:
aaah yes, that thing you mentioned... I forgot to do it. Thanks